dao/mock_daos.go

```diff
@@ -157,6 +161,10 @@ func (s *MockSourceDao) Unpause(_ int64) error {
 	return nil
 }
 
+func (s *MockSourceDao) IsPaused(_ *int64) (bool, error) {
+	return false, nil
```

Comment:
Maybe here should be return `s.IsPausedReturnValue, nil`?

Reply:
Nice catch :fish: !